webapp/src/pages/PrivateAccountsList.js

```diff
@@ -67,17 +67,17 @@ export default function PrivateAccountsList() {
     <>
       <LayoutContainer top>
         <LinearBreadcrumbs back="/dashboard" />
-        <h2>Private Accounts</h2>
-        <p className="text-secondary mt-3">
-          Used to hide your email and phone number from vendors you use through the Suma
-          platform. Follow the instructions for the vendors you want to create a Private
-          Account with.
-        </p>
+        <h2>{t("mobility:private_accounts_title")}</h2>
```

Comment:
This should be in a `private_accounts` namespace, not the `mobility` namespace. These are not tied directly to mobility, just happen to be used by certain mobility vendors.